docs/DeveloperGuide.adoc

```diff
@@ -206,7 +202,22 @@ This help feature is a functionality that is carried out by the FoodRecordParser
 With this, a top-level idea of the execution of the help command is given in the sequence diagram below:
 
 .Sequence Diagram for Help Command
-image::HelpSequenceDiagram.png[]
+image::HelpSequenceDiagram.png[]// tag::helpcommand[]
+[[Implementation-HelpGuide]]
+=== Command guide `help` command
+
+This section addresses how the `help` command works.
+
+The `help` command allows us to reference a summarised version of the User Guide (called the command guide)
+containing the usages of the commands and their formats, arranged in alphabetical order. Users may enter an
+optional `Command_word` that filters the displayed command guide.
+
+[NOTE]
+`Command_word` filters out only commands which contain the `Command_word` as a substring. If no commands contain it
```

Comment:
Hi Janice, there seems to be some issues standardising across the team. These affect readability and consistency.
However, don't worry. We will discuss them in our next meeting.

Cheers!